lib/trento/activity_logging/parser/metadata_enricher.ex

```diff
@@ -82,6 +86,9 @@ defmodule Trento.ActivityLog.Logger.Parser.MetadataEnricher do
        when activity in [:resource_tagging, :resource_untagging],
        do: {:ok, resource_id}
 
+  defp detect_enrichment(:host, {_, %{resource_id: id, operation: :saptune_solution_apply}}),
```

Comment:
Using `operation` instead of the activity name itself, as we need to differenciate between host/cluster/sap system resource at some  point.
Either way, we will see how to pattern match once we have more operations